package.json

```diff
@@ -15,7 +15,8 @@
     "gulp-rename": "^1.4.0",
     "gulp-uglify": "^3.0.1",
     "gulp-umd": "^0.2.1",
-    "gulp-util": "^3.0.8"
+    "gulp-util": "^3.0.8",
+    "natives": "^1.1.6"
```

Comment:
What is this used for?

Reply:
@stuyam I couldn't get the project to build via gulp - the error message led me to this, and adding this package was gulps recommendation (seems to have hidden my reference to it it in the commit)

https://github.com/gulpjs/gulp/issues/2246